### ISSUE TITLE
[BUG] Can no longer bring up new CRC 1.9.0

### ISSUE THREAD

**Issue by phantomjinx** [code-ready/crc#1271]
### General information

  * OS: Linux
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: Laptop

## CRC version
```bash
bin$ crc version
crc version: 1.9.0+a68b5e0
OpenShift version: 4.3.10 (embedded in binary)
```

## CRC config
```bash
- memory                                : 16384
- nameserver                            : 192.168.250.12
- skip-check-crc-dnsmasq-file           : true
- skip-check-network-manager-config     : true
```

## Host Operating System
```
bin$ cat /etc/os-release
NAME=Fedora
VERSION="30 (Workstation Edition)"
ID=fedora
VERSION_ID=30
VERSION_CODENAME=""
PLATFORM_ID="platform:f30"
PRETTY_NAME="Fedora 30 (Workstation Edition)"
ANSI_COLOR="0;34"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:30"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f30/system-administrators-guide/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=30
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=30
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation
```

### Steps to reproduce

1. downloaded/extracted crc-1.9.0
2. crc delete
3. crc setup
4. crc start

### Expected
Certificates get renewed and cluster comes up.

### Actual
Certificates get renewed but node certificate remains expired.

### Logs
debug log => https://gist.github.com/phantomjinx/56be6e81b3dfd899d945f04f2a59241d

### Note
Having shutdown 1.9.0 to upgrade to 1.10.0, I encountered #1242 so had no choice but to revert to 1.9.0. Consequently, cannot get 1.9.0 back up either.

**Comment by btannous** [comment]
v1.8 should work (from my testing before)

**Comment by gbraad** [comment]
Duplicate of https://github.com/code-ready/crc/issues/1242. We are aware of the issue and looking into this. A new release will be available soon to provide you with a working cluster with new certificates while we investigate why the renew process fails.

**Comment by phantomjinx** [comment]
@gbraad 
This is not a duplicate of #1242 as it relates to 1.9.0 and not 1.10.0. Although, errors are similar, it would appear 1.9.0 has a slightly different issue (although some people appear to be ok with it).

**Comment by gbraad** [comment]
Rather see this reported on the same issue too. Wil reopen after confirmation
